tools/mysql_to_elastic_new_topics.py

```diff
@@ -26,8 +26,8 @@
 DELETE_INDEX = False
 # First we should only index with topics so we can do
 # inference based on the articles that have topics.
-INDEX_WITH_TOPIC_ONLY = False
-TOTAL_ITEMS = 50000
+INDEX_WITH_TOPIC_ONLY = True
+TOTAL_ITEMS = 10000
```

Comment:
why is total items 10K?

Reply:
Because I didn't want to run it for all documents in my environment. I can add a comment that number should be adjusted as needed, I guess it could even be made into a system argument.